src/components/Pages/OrderPage.js

```diff
@@ -55,24 +48,38 @@ const OrderPage = () => {
           onCloseModal={modalHandler}
         />
       )}
-      <CommonContainer withMargin={true}>
-        <Wrapper>
-          <OrderContainer>
-            <OrderForm onFormChange={onFormChange}></OrderForm>
-            <ShadowContainer withShadow={"withShadow"}>
-              <OrderList
-                orderItems={cartContext.items}
-                totalAmount={cartContext.totalAmount}
-              ></OrderList>
-            </ShadowContainer>
-          </OrderContainer>
-          <BtnContainer>
-            <Button className={"order"} onClick={submitHandler}>
-              Order <Icons name="check" classes={"icon"} />
-            </Button>
-          </BtnContainer>
-        </Wrapper>
-      </CommonContainer>
+      {isLoading && <Spinner />}
+
+      {error && error.message && (
+        <Modal title={error.message} onCloseModal={modalHandler} />
+      )}
+      <Formik
+        initialValues={initialFormState}
+        validationSchema={OrderSchema}
+        onSubmit={submitHandler}
+      >
+        {({ submitForm }) => (
+          <CommonContainer withMargin={true}>
+            <Wrapper>
+              <OrderContainer>
+                <OrderForm cartContext={cartContext}></OrderForm>
```

Comment:
There's no need to pass context as props. We can access it directly in the OrderForm Component